### ISSUE TITLE
Bug: Remove `connectionRetryLimit` from documentation and options (it is not used)

### ISSUE THREAD

**Issue by titanism** [parse-community/node-apn#146]
The option `connectionRetryLimit` is not used anywhere in the codebase, but it's documented as an option:

https://github.com/search?q=repo%3Aparse-community%2Fnode-apn+connectionRetryLimit&type=code

**Comment by cbaker6** [comment]
I don't think you should remove the option, just needs to be used in the client

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.4.0](https://github.com/parse-community/node-apn/releases/tag/6.4.0)
